pages/common/gcloud-help.md

```diff
@@ -0,0 +1,9 @@
+# gcloud help
+
+> Display help and reference information for `gcloud`.
+> For supplementary help for topics not directly associated with individual commands, please take a look at `tldr gcloud topic`.
```

Comment:
Perhaps this will read better?
```suggestion
> For supplementary help for topics not directly associated with individual commands, see also `tldr gcloud topic`.
```